.github/actions/ortdocker/action.yml

```diff
@@ -15,29 +15,29 @@
 # SPDX-License-Identifier: Apache-2.0
 # License-Filename: LICENSE
 
-name: "ORT Docker image"
-description: "Check and create Docker image for ORT components"
-author: "The ORT Project Authors"
+name: 'ORT Docker image'
```

Comment:
Please add to the commit message *which* linter was used for this normalization.

Reply:
Done

Reply:
> Used RedHat VSCode YAML Language server.

I'm not sure we want this. Each linter lints differently, and VSCode is not an editor that we officially support for ORT (only IntelliJ IDEA is). Also, it seems a bit arbitrary to only "normalize" the Docker-related YAML files in the `.github` directory, and not all of them.

Personally, I'm for dropping this commit. What do @oss-review-toolkit/core-devs say?

Reply:
I dropped the linter commit.
Maybe we should agreed with some default with prettier, or similar that can be added in the code base later.